js/index.js

```diff
@@ -1 +1,110 @@
-// Your code goes here
+const headerimg = document.querySelector('header img');
+const backcolor = document.querySelector('body');
+const footercolor = document.querySelector('footer');
+const largesubtitle = document.querySelector('header h2');
+const header = document.querySelector('header');
+const links = document.querySelectorAll('nav a');
+const title = document.querySelector('h1');
+const btn = document.querySelectorAll('.btn');
+const changeColor = document.querySelectorAll('h4');
+const images = document.querySelectorAll('img')
+
```

Comment:
Coders typically have two different ways to create variables:

1. Variables created at the top, then used throughout the code
2. Variables created just a line or 2 before they're used in the code

Either way works; it's up the the developer to decide how to stay organized. I like that you've got an organization system 💪